src/components/navigation/navigation.spec.js

```diff
@@ -230,3 +290,19 @@ describe('script: navigation', () => {
     },
   );
 });
+
+describe.each([['sub', EXAMPLE_NAVIGATION_WITH_HIDDEN_SUBNAVIGATION, '.ons-navigation--sub']])(
+  'level: %s navigation',
+  (_, params, navEl) => {
```

Comment:
Do we need the `navEl` var here considering we are only using it in one place?

Reply:
We are using it here, should it be removed?
  it('removes the sub-nav element', async () => {
        const hasSubNavEl = (await page.content()).includes(navEl);
        expect(hasSubNavEl).toBe(false);
      });
    });

Reply:
Im just saying do we need it as a variable? Its only used in one place so we could just set it there instead of passing it in

Reply:
We are still passing the '.ons-navigation--sub' class in on line 294. It should be removed from there too